app/views/curation_concerns/base/_relationship.html.erb

```diff
@@ -0,0 +1,36 @@
+<h3><% if presenters.first.citi_presenter? %>
```

Comment:
I think it would be better to have two kinds of views, one for rel_term and another for rel_term_asset, and have the helper methods all the appropriate one. It avoids have to duplicate this logic. However, if this is doing what you need, we can go with this for now.